tests/test_io.py

```diff
@@ -84,6 +97,7 @@ def test_get_filename_01(self):
         """
         Testing get_filename wrapper for get_text.
         """
+        dfastmi.io.load_program_texts("dfastmi/messages.UK.ini")
```

Comment:
you can add the setup_data in this test and remove the"dfastmi.io.load_program_texts("dfastmi/messages.UK.ini")"

Reply:
will resolve in https://issuetracker.deltares.nl/browse/DFAST-86